core/server/services/auth/setup.js

```diff
@@ -77,18 +77,29 @@ async function doSettings(data, settingsAPI) {
     const context = {context: {user: data.user.id}};
     const user = data.user;
     const blogTitle = data.userData.blogTitle;
+    const description = data.userData.description;
 
     let userSettings;
 
     if (!blogTitle || typeof blogTitle !== 'string') {
         return user;
     }
 
+    if (!description || typeof description !== 'string') {
+        return user;
+    }
```

Comment:
I think this will cause an issue: a setup call with a title but without a description will prevent the title from being set. I would remove the IF completely (unless I've missed something).

Reply:
This should be good now! Thanks for checking @tpatel 🤗